airflow/providers/google/cloud/hooks/dataprep.py

```diff
@@ -66,10 +67,57 @@ def get_jobs_for_job_group(self, job_id: int) -> Dict[str, Any]:
         """
         Get information about the batch jobs within a Cloud Dataprep job.
 
-        :param job_id The ID of the job that will be fetched.
+        :param job_id The ID of the job that will be fetched
         :type job_id: int
         """
+
         url: str = f"{self._url}/{job_id}/jobs"
         response = requests.get(url, headers=self._headers)
         response.raise_for_status()
         return response.json()
+
+    @retry(stop=stop_after_attempt(5), wait=wait_exponential(multiplier=1, max=10))
+    def get_job_group(self, job_id: int, embed: str, include_deleted: bool) -> Dict[str, Any]:
+        """
+        Get the specified job group.
+        A job group is a job that is executed from a specific node in a flow.
+
+        :param job_id The ID of the job that will be fetched
+        :type job_id: int
+        :param embed Comma-separated list of objects to pull in as part of the response
+        :type embed: str
+        :param include_deleted if set to "true", will include deleted objects
+        :type include_deleted: bool
+        """
+
+        params: Dict[str, Any] = {
+            'embed': embed,
+            'includeDeleted': include_deleted
+        }
+
+        url: str = f"{self._url}/{job_id}"
```

Comment:
You should define these variables with base_url, which should be configurable and default to ``https://api.clouddataprep.com/`` and endpoint name. - In this cases ``v4/jobGroups``
```
        sellf._base_url = self.conn.extra_dejson.get('base_url', 'https://api.clouddataprep.com/')
```
```
       endpoint_path = '/v4/jobGroups/{job_groupp__id}'
        url: str = f"{self._url}/{endpoint_path}"
```
Otherwise, it's very confusing as it's hard to figure out which endpoint you are using.

Reply:
Do you mean property @mik-laj?
Like this?

```
@property
    def _base_url(self) -> str:
        conn = self.get_connection(self.dataprep_conn_id)
        url = conn.extra_dejson.get("base_url", "https://api.clouddataprep.com/")
        if url is None:
            raise AirflowException(
                "Dataprep base url is missing or has invalid format. "
                "Please make sure that Dataprep url is added to the Airflow Connections."
            )
        return url
```

Reply:
Discussed via slack.